docs/components/using-other-components.md

```diff
@@ -0,0 +1,90 @@
+# Using other components
```

Comment:
Maybe this should be "migrating to components"?  There's already a "Using Components" (`using.md`).